docs/docs/new-features.md

```
@@ -180,6 +180,38 @@ Everyone sees the margins and follows the presenter's point of view. If multi-us
 Recording is not yet implemented, meaning that if you enable this experimental feature on your server and use it in a recorded session, the recording will most likely have broken whiteboard at best. The recording (and playback) work is planned for after BigBlueButton 3.0.
 
+#### Integration with LiveKit
+
+We have added initial support for LiveKit as a media framework for BigBlueButton.
+It's an experimental feature and, consequently, disabled by default.
+For an in-depth overview of this initiative, please refer to [issue 21059](https://github.com/bigbluebutton/bigbluebutton/issues/21059).
+Feature parity with the current media framework is not yet achieved, but the
+aforementioned issue provides parity tracking in section `Annex 1`.
+
+To enable support for LiveKit:
+  - Install bbb-livekit: `$ sudo apt-get install bbb-livekit`
```

Markdownlint (0.35.0)

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)